OmniKitUI/ViewControllers/ReplacePodViewController.swift

```diff
@@ -20,19 +20,36 @@ class ReplacePodViewController: SetupTableViewController {
         case canceledPairing
     }
 
+    private var faultType: FaultEventCode.FaultEventType?
```

Comment:
It'd be a little cleaner to make the PodReplacementReason.fault case contain the fault type.